tutorial/source/gmm.ipynb

```diff
@@ -144,15 +151,15 @@
BcJBH/qv/Wd/IChvICh/eH/bz63nhm4YjLyBoIKRTwu8AAAIjklEQVTGyB8woZ9Bo0AwtB0IBhVo3Rc4eSxg2m4H5Q8a5fRL7PU6aNWjtT9nzhxt2bJFlZWVys3N1SOPPKL58+dr2bJlmj59ugKBgObNm6fRo0d36bozZ87UzJkzGYMHALQTakk7WHtAks2Y3urQ6H6FhYX64IMPYl0MAAB6RVdyjwckAQCIQ31mFn1XMAYPAMCZWbIFP3PmTJWWlio9PT3WRQEAoE+yZMADAIAzI+ABAIhDBDwAAHGISXYAAMQhSz8HP3DgQA0bNqzbrnfs2DENGjSo2653oaIezx91eP6ow+5BPZ6/7qzD8vJyVVZWRnWupQO+u7FwTvegHs8fdXj+qMPuQT2ev1jVIWPwAADEIQIeAIA45Hj44YcfjnUh+pLLL7881kWIC9Tj+aMOzx912D2ox/MXizpkDB4AgDhEFz0AAHGIgAcAIA4R8GEbN27UJZdcooKCAj3xxBOxLk6fNW/ePGVmZmrMmDGRfcePH9d1112niy++WNddd51OnDghSTLG6N5771VBQYHGjRunnTt3xqrYfUpFRYWuueYajRo1SqNHj9bTTz8tiXrsqubmZk2aNEnjx4/X6NGj9R//8R+SpP3792vy5MkqKCjQ17/+dXm9XklSS0uLvv71r6ugoECTJ09WeXl5DEvftwQCAV122WX6yle+Iok67Kphw4Zp7NixmjBhggoLCyX1kb9nA+P3+01+fr7Zt2+faWlpMePGjTN79uyJdbH6pLffftv8+c9/NqNHj47sW7RokVm6dKkxxpilS5ea733ve8YYY9avX2+uv/56EwwGzbZt28ykSZNiUua+5siRI+bPf/6zMcaY2tpac/HFF5s9e/ZQj10UDAZNXV2dMcYYr9drJk2aZLZt22Zuvvlms3r1amOMMd/61rfMf//3fxtjjHn22WfNt771LWOMMatXrza33HJLbAreB/30pz81c+bMMTNmzDDGGOqwi4YOHWqOHTvWbl9f+Hsm4I0x7777rpk2bVpk+/HHHzePP/54DEvUt+3fv79dwI8YMcIcOXLEGBMKrxEjRhhjjCkpKTGrVq3q9DycVFxcbDZt2kQ9noeGhgZz2WWXmffee89kZGQYn89njGn/tz1t2jTz7rvvGmOM8fl8JiMjwwSDwZiVua+oqKgwU6dONW+99ZaZMWOGCQaD1GEXdRbwfeHvmS56SYcPH1ZeXl5kOzc3V4cPH45hiazl6NGjGjJkiCQpKytLR48elUS9RqO8vFwffvihJk+eTD2eg0AgoAkTJigzM1PXXXedhg8frn79+snpDL1mo21dta1Hp9Op9PR0VVVVxazsfcV9992nH/3oR7LbQ3FQVVVFHXaRzWbTtGnTdPnll6u0tFRS3/j/oiVfNoO+y2azyWazxboYllBfX6+bbrpJTz31lNLS0todox6j43A4tGvXLlVXV+vGG2/U3/72t1gXyVLeeOMNZWZm6vLLL9eWLVtiXRzLeuedd5STk6PPPvtM1113nS699NJ2x2P190wLXlJOTo4qKioi24cOHVJOTk4MS2QtgwcP1qeffipJ+vTTT5WZmSmJej0Tn8+nm266Sbfddpu+9rWvSaIez0e/fv10zTXXaNu2baqurpbf75fUvq7a1qPf71dNTY0yMjJiVua+4E9/+pN+97vfadiwYZo9e7b+8Ic/aOHChdRhF7XWT2Zmpm688Ubt2LGjT/w9E/CSioqKtHfvXu3fv19er1dr1qxRcXFxrItlGcXFxXrhhRckSS+88IL+9V//NbL/xRdflDFG7733ntLT0yNdVhcyY4zmz5+vkSNH6rvf/W5kP/XYNceOHVN1dbUkqampSZs3b9bIkSN1zTXX6LXXXpN0aj221u9rr72mqVOnXvC9JEuXLtWhQ4dUXl6uNWvWaOrUqfr1r39NHXZBQ0OD6urqIp83bdqkMWPG9I2/5x4Z2beg9evXm4svvtjk5+ebJUuWxLo4fdbs2bNNVlaWcTqdJicnxzz//POmsrLSTJ061RQUFJgvfelLpqqqyhgTmuX8ne98x+Tn55sxY8aY999/P8al7xu2bt1qJJmxY8ea8ePHm/Hjx5v169dTj130l7/8xUyYMMGMHTvWjB492jzyyCPGGGP27dtnioqKzPDhw82sWbNMc3OzMcaYpqYmM2vWLDN8+HBTVFRk9u3bF8vi9zl//OMfI7PoqcPo7du3z4wbN86MGzfOjBo1KpIffeHvmaVqAQCIQ3TRAwAQhwh4AADiEAEPAEAcIuABAIhDBDwAAHGIgAcQMWXKFEmhJXRXrVoV49IAOB8EPICId999V9K5BXzrymcA+gYCHkBESkqKJGnx4sXaunWrJkyYoJ/97GcKBAJatGiRioqKNG7cOP385z+XJG3ZskVXXXWViouLNWrUKDU0NGjGjBkaP368xowZo1deeSWW/xzggsbLZgCc4oknntBPfvITvfHGG5Kk0tJSpaen6/3331dLS4u+8IUvaNq0aZKknTt36q9//asuuugi/fa3v1V2drbWr18vSaqpqYnZvwG40NGCB3BWmzZt0osvvqgJEyZo8uTJqqqq0t69eyVJkyZN0kUXXSRJGjt2rDZv3qzvf//72rp1q9LT02NZbOCCRsADOCtjjJ555hnt2rVLu3bt0v79+yMt+OTk5Mh5I0aM0M6dOzV27Fg9+OCDevTRR2NVZOCCR8ADOEVqamrkDVmSNH36dC1fvlw+n0+S9Pe//10NDQ2nfO/IkSNKSkrSN77xDS1atEg7d+7stTIDaI8xeACnGDdunBwOh8aPH6+5c+dq4cKFKi8v18SJE2WM0aBBg7R27dpTvve///u/WrRokex2u1wul5YvXx6D0gOQJN4mBwBAHKKLHgCAOETAAwAQhwh4AADiEAEPAEAcIuABAIhDBDwAAHGIgAcAIA79f6Gk+m1wMWKQAAAAAElFTkSuQmCC\n",
```

Comment:
would be nice to save to file and just embed the image here, like in vae tutorials

Reply:
What's the advantage of that? Note that I cannot see the VAE tutorial images locally when I use jupyter (maybe due to path errors).